frontend/src/general/__tests__/DeleteDeviceButton.test.jsx

```diff
@@ -91,64 +79,60 @@ describe("DeleteButton", () => {
     });
 
     describe("ConfirmButton", () => {
-      describe("on DeviceListPage", () => {
-        it("Should call axios.delete, close the dialog and refresh the page when clicked", async () => {
-          // Pretend we are on the device list page
-          mockHistory.location.pathname = "/Devices";
-
-          // click the delete button to set open to true
-          wrapper.find("#deleteBtn").simulate("click");
-          expect(setOpen).toHaveBeenCalledWith(true);
+      it("Should call axios.delete, close the dialog and display a success snackbar", async () => {
+        // click the delete button to set open to true
+        wrapper.find("#deleteBtn").simulate("click");
+        expect(setOpen).toHaveBeenCalledWith(true);
 
-          // mock axios before clicking confirm
-          const axiosPromise = Promise.resolve();
-          axios.delete.mockImplementationOnce(() => axiosPromise);
+        // mock axios before clicking confirm
+        axios.delete.mockResolvedValueOnce();
 
-          // click confirm
-          wrapper.find("#confirmDeleteBtn").simulate("click");
+        // click confirm
+        wrapper.find("#confirmDeleteBtn").simulate("click");
 
-          // check that axios.delete was called
-          expect(axios.delete).toHaveBeenCalled();
+        // check that axios.delete was called
+        expect(axios.delete).toHaveBeenCalled();
```

Comment:
Comments like this are kind of redundant imo, i'd remove them. The whole point of jest's `expect` is that you can read the assert line like english "expect axios.delete to have been called" so they don't need a comment explaining what they're doing.

Reply:
I added the comment cause some of the tests had comments like this, but I'll remove it!